### ISSUE TITLE
Nansat lectrues installation 

### ISSUE THREAD

**Issue by Ghazal87** [nansencenter/nansat-lectures#6]
I tried to install the lectures and I encountered this problem after this line 

vagrant up course

Bringing machine 'course' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

ansible local provisioner:
* The following settings shouldn't exist: install_mode

**Comment by akorosov** [comment]
Which vagrant version do you use? It is probably outdated.
Docs: https://www.vagrantup.com/docs/provisioning/ansible_local.html

**Comment by Ghazal87** [comment]
I have this version 1.8.1+dfsg-1

**Comment by Ghazal87** [comment]
it is Vagrant 1.8.1

**Comment by akorosov** [comment]
Try updating to 1.9.1

**Comment by Ghazal87** [comment]
I updated the vagrant to 1.9.1 and again run vagrant up course and I encountered this error.

vagrant up course
Bringing machine 'course' up with 'virtualbox' provider...
==> course: Checking if box 'ubuntu/trusty64' is up to date...
==> course: Clearing any previously set forwarded ports...
==> course: Clearing any previously set network interfaces...
==> course: Preparing network interfaces based on configuration...
    course: Adapter 1: nat
    course: Adapter 2: hostonly
==> course: Forwarding ports...
    course: 22 (guest) => 2222 (host) (adapter 1)
==> course: Running 'pre-boot' VM customizations...
==> course: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "5e076343-05d5-4c55-84ad-b80830aecdab", "--type", "headless"]

Stderr: VBoxManage: error: VT-x is not available (VERR_VMX_NO_VMX)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
